docs/changelog.md

```diff
@@ -10,10 +10,13 @@ permalink: /changelog/
 * [Gecko](https://github.com/mozilla-mobile/firefox-android/blob/main/android-components/plugins/dependencies/src/main/java/Gecko.kt)
 * [Configuration](https://github.com/mozilla-mobile/firefox-android/blob/main/android-components/.config.yml)
 
+* **lib-crash-sentry**
+  * Adds paramter to initialize Sentry with `SentryService`.
```

Comment:
Typo:

```suggestion
  * Adds parameter to initialize Sentry with `SentryService`.
```

(sorry I only stumbled upon this patch accidentally, my OCD did the rest..)

Reply:
thank you!